src/ucp/core/ucp_mm.c

```diff
@@ -540,6 +553,19 @@ static ucs_status_t ucp_memh_create_from_mem(ucp_context_h context,
     return UCS_OK;
 }
 
+static UCS_F_ALWAYS_INLINE void
+ucp_memh_init_from_parent(ucp_mem_h memh, ucp_md_map_t parent_md_map)
+{
+    ucp_md_index_t md_index;
+
+    memh->reg_id  = memh->parent->reg_id;
+    memh->md_map |= memh->parent->md_map;
```

Comment:
we set `memh->parent->md_map`, but iterate and take uct memhs by using another map  `parent_md_map`. 
this can create an inconsistency between memh->map_map and UCT mem handles

Reply:
fixed